srcpkgs/ntfs-3g/template

```diff
@@ -1,22 +1,24 @@
 # Template file for 'ntfs-3g'
 pkgname=ntfs-3g
-version=2017.3.23
-revision=2
-wrksrc="${pkgname}_ntfsprogs-${version}"
+version=2021.8.22
+revision=1
+wrksrc="ntfs-3g_ntfsprogs-${version}"
```

Comment:
Avoiding `$pkgname` in `$wrksrc` is unnecessary, we only want to avoid it in `$homepage` and `$distfiles`.

Reply:
In general it is less strict, I would (1) leave usage (no usage) as is except homepage, (2) use variable where it _means_ variable. That is: assume you can't use preferred pkgname. Would you use preferred, or actual pkgname there? If actual, use variable, otherwise spell out.